kwargs.jl

```diff
@@ -5,14 +5,21 @@ import Pkg
 include(joinpath(@__DIR__, "autodetect-dependabot.jl"))
 
 function kwargs(; coverage::Bool,
-                  force_latest_compatible_version::Union{Bool, Symbol})
+                  force_latest_compatible_version::Union{Bool, Symbol},
+                  julia_args::Union{Cmd, AbstractVector{<:AbstractString}}=``)
     if !(force_latest_compatible_version isa Bool) && (force_latest_compatible_version != :auto)
         throw(ArgumentError("Invalid value for force_latest_compatible_version: $(force_latest_compatible_version)"))
     end
 
     kwargs_dict = Dict{Symbol, Any}()
     kwargs_dict[:coverage] = coverage
 
+    if VERSION >= v"1.6.0"
+        kwargs_dict[:julia_args] = julia_args
+    elseif julia_args != ``
+        println("::warning::The `julia_args` option requires at least Julia 1.6. VERSION=$VERSION, julia_args=$julia_args")
+    end
```

Comment:
Sorry to ask again: Can `julia_args` ever be ` `` ` here? It's set to `julia_args = ["--check-bounds=${{ inputs.check_bounds }}"]` in `action.yml` and `${{ inputs.check_bounds }}` defaults to `yes`.

Reply:
Good catch. Should be fixed now. The change makes kwargs.jl more specific to how it's used in this action, which I assumed wasn't a problem